### ISSUE TITLE
A1CN very slow

### ISSUE THREAD

**Issue by A-Maugli** [AustP/austs-one-click-node#39]
On a PC with a 8-core CPU, 16 GB, under WSL2 the algod sync takes about 5500 sec. 
Under Win10 A1CN V1.5.0 is now running for 14086 sec, and still not ready. Is it normal?
Why is the Win10 version so slow, compared to WSL2 algod?

**Comment by AustP** [comment]
No, that doesn't seem right. As to why that happened, I cannot say. Things you could try are restarting the app or reinstalling the app.

**Comment by A-Maugli** [comment]
The issue may be connected with high GPU 3D usage, typically around 30
percent. My PC has an NVidia GeForce GTX 1650.
When I started the A1CN, it began to synchronize again, although yesterday
the mainnet was already synced.

**Comment by A-Maugli** [comment]
I have reinstalled the A1CN app, and spun up Algorand mainnet. But this time I kept the app in minimised state, and it run a little faster, under 12000 sec. What else do you recommend to try?

**Comment by AustP** [comment]
Sometimes there's a cache file that gets messed up. Turning off the app and then deleting the folder and then starting the app will cause a long resync but then it is good after that. The folder to delete is based on your OS:

Windows: "C:\Users\username\AppData\Roaming\Austs One-Click Node\data\algorand.mainnet"
Mac: "/Users/username/Library/Application Support/Austs One-Click Node/data/algorand.mainnet"
Linux: "/home/username/.config/Austs One-Click Node/data/algorand.mainnet"
